pom.xml

```diff
@@ -24,7 +24,7 @@
 
   <properties>
     <antlr4.version>4.7.2</antlr4.version>
-    <drools.version>7.0.0.Final</drools.version>
+    <drools.version>7.53.0.Final</drools.version>
```

Comment:
The last time an upgrade to Drools was attempted it caused an outage to the hosted reference environments, due to it not working in FOLIO's docker images.

Has this been tested in a deployed environment using Docker?

Reply:
Thanks for mentioning this, I haven't seen see the comment in the README.

I can reproduce the NullPointerException issue locally and added a fix for it to this PR.

Reply:
Cool, thank you for figuring this out and fixing it